plugins/generic/googleScholar/GoogleScholarPlugin.inc.php

```diff
@@ -0,0 +1,138 @@
+			$templateMgr->addHeader('googleScholarIssn', '<meta name="citation_issn" content="' . htmlspecialchars($issn) . '"/> ');
+			
+		}
+
+		foreach ($article->getAuthors() as $i => $author) {
+			$templateMgr->addHeader('googleScholarAuthor' . $i, '<meta name="citation_author" content="' . htmlspecialchars($author->getFirstName()) . (($middleName = htmlspecialchars($author->getMiddleName()))?" $middleName":'') . ' ' . htmlspecialchars($author->getLastName()) . '"/>');
+			if ($affiliation = htmlspecialchars($author->getLocalizedAffiliation())) {
+				$templateMgr->addHeader('googleScholarAuthor' . $i . 'Affiliation', '<meta name="citation_author_institution" content="' . $affiliation . '"/>');
+			}
+		}
+
+		$templateMgr->addHeader('googleScholarTitle', '<meta name="citation_title" content="' . htmlspecialchars($article->getLocalizedTitle()) . '"/>');
+
+		if (is_a($article, 'PublishedArticle') && ($datePublished = $article->getDatePublished())) {
+			$templateMgr->addHeader('googleScholarDate', '<meta name="citation_date" content="' . strftime('%Y/%m/%d', strtotime($datePublished)) . '"/>');
+		} elseif ($issue && $issue->getYear()) {
+			$templateMgr->addHeader('googleScholarDate', '<meta name="citation_date" content="' . htmlspecialchars($issue->getYear()) . '"/>');
+		} elseif ($issue && ($datePublished = $issue->getDatePublished())) {
+			$templateMgr->addHeader('googleScholarDate', '<meta name="citation_date" content="' . strftime('%Y/%m/%d', strtotime($datePublished)) . '"/>');
+		}
+
+		if ($issue) {
+			$templateMgr->addHeader('googleScholarVolume', '<meta name="citation_volume" content="' . htmlspecialchars($issue->getVolume()) . '"/>');
+			$templateMgr->addHeader('googleScholarNumber', '<meta name="citation_issue" content="' . htmlspecialchars($issue->getNumber()) . '"/>');
+		}
+
+		if ($article->getPages()) {
+			if ($startPage = $article->getStartingPage()) $templateMgr->addHeader('googleScholarStartPage', '<meta name="citation_firstpage" content="' . htmlspecialchars($startPage) . '"/>');
+			if ($endPage = $article->getEndingPage()) $templateMgr->addHeader('googleScholarEndPage', '<meta name="citation_lastpage" content="' . htmlspecialchars($endPage) . '"/>');
+		}
+
+		foreach($templateMgr->get_template_vars('pubIdPlugins') as $pubIdPlugin) {
+			if ($pubId = $pubIdPlugin->getPubId($article)) {
+				$templateMgr->addHeader('googleScholarPubId' . $pubIdPlugin->getPubIdDisplayType(), '<meta name="citation_' . htmlspecialchars(strtolower($pubIdPlugin->getPubIdDisplayType())) . '" content="' . htmlspecialchars($pubId) . '"/>');
+				
+			}
+		}
+
+		$templateMgr->addHeader('googleScholarHtmlUrl', '<meta name="citation_abstract_html_url" content="' . $request->url(null, 'article', 'view', array($article->getBestArticleId($journal))) . '"/>');
+		if ($language = $article->getLanguage()) $templateMgr->addHeader('googleScholarLanguage', '<meta name="citation_language" content="' . htmlspecialchars($language()) . '"/>');
```

Comment:
From http://www.monperrus.net/martin/accurate+bibliographic+metadata+and+google+scholar:

> __What about the other citation tags?__ There are many of them in the wild. The tags that aren't listed in the guidelines are not not officially supported, so their effect on indexing in Google Scholar should be considered undefined (information given by Google Scholar Support): citation_date, citation_conference, citation_doi, citation_abstract_html_url, citation_fulltext_html_url, citation_publisher, __citation_language__, citation_pmid, citation_keywords, citation_dissertation_name, citation_patent_number, citation_patent_country.

I just took the existing behavior, and while that GS meta element isn't documented, I'm hesitant to change it. GS probably has profiles on data providers e.g. by application.